tests/unit/Framework/MockObject/Creation/CreateMockTest.php

```diff
@@ -53,12 +52,13 @@ public function testCannotCreateMockObjectForFinalClass(): void
         $this->createMock(FinalClass::class);
     }
 
-    #[RequiresPhp('8.2')]
```

Comment:
The coverage pipeline was running using PHP 8.3 since release, but it was not working since this mentioned 8.2 exactly.